### PR TITLE
agent workflow update

### PR DESCRIPTION
**Pull request by snova-luiss**

### CODE REVIEW

**Review by snova-codym — APPROVED** [review]
I still need to best understand the API gateway, but this change looks reasonable.

File: agent_workflows/config.yaml [hunk 1]
```diff
@@ -1,16 +1,23 @@
...
     "temperature": 0.0
-    "max_tokens_to_generate": 1024
-    "sambaverse_model_name": "Meta/llama-3-70b-chat-hf"
-    "sambaverse_select_expert": "Meta-Llama-3-70B-Instruct"
+    "do_sample": False
+    "max_tokens_to_generate": 1200
+    "sambaverse_model_name": "Meta/Meta-Llama-3-8B-Instruct"
+    "coe": True #set as true if using Sambastudio CoE endpoint
+    "select_expert": "Meta-Llama-3-8B-Instruct" #set if using FastCoE, Sambaverse or SambaStudio CoE llm expert
+    #fastApi CoE expert name -> "llama3-8b"
```
Comment (snova-kwasia) [inline agent_workflows/config.yaml:20]: Any reason this is commented out @snova-luiss ? Thanks

**Review by snova-kwasia — APPROVED** [review]
Looks good to me, minor comment added.

File: agent_workflows/config.yaml (hunk 1 above)
Comment (snova-luiss) [reply agent_workflows/config.yaml:20]: Not code, it’s just the continuation of the instruction from the previous line

File: agent_workflows/prompts/llama3-prompt_engineering-final_chain.yaml [hunk 2]
```diff
@@ -11,7 +11,7 @@ template: |
     You are an assistant for question-answering tasks.
     Use the following generation to provide a final answer to the original question.  If you cannot answer based on the generation provided to you, say "I apologize, but I cannot provide the final answer due to a lack of grounding in documents or an ability to generate and run code that is required to solve the task".  Please do not comment on how to obtain further information about partially answered questions.  If there is specific advice that could help the user find information from the provided generation, do provide all of this advice as part of your answer/apology.<|eot_id|><|start_header_id|>user<|end_header_id|>
     Original Question: {question} 
-    Generation: {generation}
```
Comment (snova-codym) [inline agent_workflows/prompts/llama3-prompt_engineering-final_chain.yaml:14]: call_rag methods expect generation.  Here is an example:

https://github.com/sambanova/ai-starter-kit/blob/7e9c9477c2235d444b1b59954a3841c1ccb6df8d/utils/agents/static_RAG_with_coding.py#L246

**Review by snova-codym — APPROVED** [review]
Thanks!

**Comment by snova-rodrigom** [comment]
thanks everyone!

**Pull request merged**